basyx.aasrepository/basyx.aasrepository-backend-mongodb/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/TestMongoDBAasRepository.java

```diff
@@ -35,30 +35,35 @@
 import org.eclipse.digitaltwin.basyx.common.mongocore.MongoDBUtilities;
 import org.junit.Test;
 import org.springframework.data.mongodb.core.MongoTemplate;
-
 import com.mongodb.client.MongoClient;
 import com.mongodb.client.MongoClients;
 
 /**
  * 
- * @author schnicke, danish
+ * @author schnicke, danish, kammognie
  *
  */
 public class TestMongoDBAasRepository extends AasRepositorySuite {
-
 	private final String COLLECTION = "aasTestCollection";
 
+	private static final String CONFIGURED_AAS_REPO_NAME = "configured-aas-repo-name";
+
 	@Override
 	protected AasRepositoryFactory getAasRepositoryFactory() {
-		String connectionURL = "mongodb://mongoAdmin:mongoPassword@localhost:27017/";
-		MongoClient client = MongoClients.create(connectionURL);
-		MongoTemplate template = new MongoTemplate(client, "BaSyxTestDb");
+		MongoTemplate template = createMongoTemplate();
 
 		MongoDBUtilities.clearCollection(template, COLLECTION);
 
 		return new MongoDBAasRepositoryFactory(template, COLLECTION, new InMemoryAasServiceFactory());
 	}
 
+	private MongoTemplate createMongoTemplate() {
```

Comment:
Please recall the access modifier's order specified in the previous comment. Hence please move this method to the end.

Reply:
done